dclab/features/emodulus/viscosity.py

```diff
@@ -242,7 +253,8 @@ def get_viscosity_mc_pbs_herold_2017(
     return eta
 
 
-def get_viscosity_water_kestin_1978(temperature: float = 23.0):
+def get_viscosity_water_kestin_1978(
+        temperature: float = 23.0) -> float | npt.NDArray:
```

Comment:
If the input type is a float here, then the output can only be a float, and not an ndarray.
However, in `get_viscosity` above the `temperature` argument can be a float or an ndarray. Is this correct?